### PR TITLE
Dom manipulation

### PR DESCRIPTION
**Pull request by ice949**
## Portfolio: details popup window
In this project we did the following
- Displayed the cards dynamically
- enabled interactions with the buttons for open and close popup window 
- Used javascript objects to store card data

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -541,6 +527,101 @@ footer {
...
+}
+
+.modal-header .close-button {
+  cursor: pointer;
+  border: none;
+  outline: none;
+  background: none;
+  font-size: 35px;
+  font-weight: bold;
+  color: #fff;
```
Comment (sja-thedude) [inline css/style.css:578]: Mobile popup:


- [ ] Kindly make sure that the background of the popup is transparent following the same colour as the Figma design.
- [ ] Make sure the background of the portfolio is blurred when the popup opens as per the Figma design.
- [ ] Kindly follow the button size as per the Figma design.
- [ ] Kindly add space between paragraph and technologies as per the Figma design.
- [ ] Kindly make sure to display the image as per the Figma design. Right now as you can see the image is not displayed.
- [ ] Kindly follow the font size , font weight for the entire popup as per the Figma design


Figma design:
Screenshot 2022-05-11 at 5 53 10 PM

Your design:
Screenshot 2022-05-11 at 6 05 23 PM

File: css/style.css [hunk 2]
```diff
@@ -541,6 +527,101 @@ footer {
...
+
+.modal-header {
+  padding: 5px 15px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-right: 0.5%;
+  width: 100%;
+}
+
```
Comment (sja-thedude) [inline css/style.css:561]: Desktop Popup: 

- [ ] Kindly make sure that the background of the popup is transparent following the same colour as the Figma design.
- [ ] Make sure the background of the portfolio is blurred when the popup opens as per the Figma design.
- [ ] Kindly reduce the space between buttons as per the Figma design.
- [ ] Kindly follow the button size as per the Figma design.
- [ ] Kindly add space between paragraph and technologies as per the Figma design.
- [ ] Kindly make sure to display the image as per the Figma design. Right now as you can see the image is not displayed.

Figma design:
Screenshot 2022-05-11 at 5 52 24 PM

Your design:
Screenshot 2022-05-11 at 6 05 01 PM

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @ice949, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Great job on implementing the popup
✔️ All linters passing
✔️ Good PR description

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/main.js [hunk 3]
```diff
@@ -19,4 +22,130 @@ navLinks.forEach((element) => {
...
+    heading: 'Multi-Post Stories',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    description2: `Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+        Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,
+        when an unknown printer took a galley of type and scrambled it 1960s `,
+    languages: ['css', 'HTML', 'Bootstrap', 'Ruby'],
+    languages2: ['css', 'HTML', 'Ruby on Rails', 'Github'],
+
+    button: 'See project',
+    image: '/images/popup.png',
```
Comment (zieeco) [inline js/main.js:54]: - [ ] **[REQUIRED CHANGES]**
- As required by the previous reviewer, Please kindly fix the path issue with your image so they can show. right now, your images are broken. You can fix it like this:
`'./images/popup.png,`

Comment (zieeco) [reply js/main.js:54]: **[OPTIONAL]**

- Please, kindly remove the `.vscode` file from your repo as its redundant regarding this project. it should be added in the `.gitignore` file.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @ice949 


Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the changes, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
_Check the comments under the review._

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/main.js (hunk 3 above)
Comment (ice949) [reply js/main.js:54]: @zieeco  @sja-thedude  But the images are [displaying]([url](url)) on my side so I don't know why they are not displaying  on your side
Screenshot from 2022-05-11 20-02-52
Screenshot from 2022-05-11 20-03-02

**Review by Wes-Isaac — APPROVED** [review]
Hi @ice949, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ice949** [comment]
@Wes-Isaac  Thanks for the review

**Pull request merged**